src/persist/src/indexed/encoding.rs

```diff
@@ -235,12 +235,12 @@ pub struct BlobUnsealedBatch {
 /// put multiple small batches in a single blob but also break a very large
 /// batch over multiple blobs. We also may want to break the latter into chunks
 /// for checksum and encryption?
-#[derive(Clone, Debug, Eq, PartialEq)]
+#[derive(Clone, Debug)]
 pub struct BlobTraceBatch {
     /// Which updates are included in this batch.
     pub desc: Description<u64>,
     /// The updates themselves.
-    pub updates: Vec<((Vec<u8>, Vec<u8>), u64, isize)>,
+    pub updates: Vec<ColumnarRecords>,
```

Comment:
let's delete the `TODO: ideally, these would be a single ColumnarRecords instead of multiple.` on BlobUnsealedBatch while you're in here. it's no longer relevant